aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -186,60 +186,41 @@ impl TransactionGasParameters {
         }
     }
 
-    /// New formula to charge storage fee for write set items based on fixed APT costs.
-    pub fn calculate_write_set_storage_fee<'a>(
-        &self,
-        ops: impl IntoIterator<Item = (&'a StateKey, &'a WriteOp)>,
-    ) -> Fee {
+    /// New formula to charge storage fee for a write, measured in APT.
+    pub fn storage_fee_per_write(&self, key: &StateKey, op: &WriteOp) -> Fee {
```

Comment:
How about "per_state_write"? Was simply using "per write" as a shorter version of it.

Not a native speaker either, but if ChatGPT is fine with it, I guess it's good enough XD
> The phrase "per write" is typically used in the context of computer storage and refers to the cost or amount of data that is written to a storage device per operation.
>
> In simple terms, it refers to the amount of data that can be written to a storage device in one single write operation. For example, if a storage device has a "per write" capacity of 1 megabyte, then it can write up to 1 megabyte of data at once.
> 
> The "per write" metric is important because it can impact the speed and efficiency of data transfer and storage. Higher "per write" capacities can allow for faster data transfers and more efficient use of storage resources.
> 
> Overall, the phrase "per write" is a technical term used to describe the capabilities and limitations of computer storage devices, particularly in terms of the amount of data that can be written to the device in a single operation.

Reply:
~~Now that ChatGPT called, I'm fine.~~
Only a nit. LGTM otherwise. @gregnazario to accept.